### ISSUE TITLE
In-Flight FFT Based Harmonic Notch Setup more examples and detailed instructions especially for large copters

### ISSUE THREAD

**Issue by firebblade** [ArduPilot/ardupilot_wiki#4403]
[In-Flight FFT Based Harmonic Notch Setup](https://ardupilot.org/copter/docs/common-imu-fft.html) - Describe the issue/suggestion and improve the title. Please keep a link to the original article if relevant.

Hello all,

I've build my second copter which is a large hexaX , enabled the HNTCH in FFT,  lift it in the air and in 2 seconds (a meter high) it started to wobble and with some lucky maneuvers managed to land it safe and sound.

I started reading the forums and most of the FFT issues were from people with large copters, 6 or 8 motors.  

As a newbie i found the docs for harmonic notch FFT hard to understand, the links put you in a "cyclic redundancy"  and are meant for small fpv quads.

I would like to request a more detailed step by step procedure with more info of what each thing should be and how its graph should look in good and bad occasions also i find it crucial to mention settings for big copters with lots of jumping around noise. 

Obviously am keen to write down what i've learned and what's in the forum, if someone could supervise it .

Thank you all for ArduPilot is truly brilliant work !!!

Best wishes!

**Comment by firebblade** [comment]
Hello Henry , 

I hope i was not misunderstood am not here to complain about my copter i just wanted to bring to your attention that the FFT instructions are hard to follow and not complete, i also wanted to state that am willing to help in making it friendlier  to new users like myself.

 As you said incorrect setup was the issue for instability but i want to attribute that in not easy to follow instructions , especially for large copter . e.g. `INS_HNTCH_OPTS == 2`  is a must to have for big copters and its not in here : https://ardupilot.org/copter/docs/common-imu-fft.html

Its here :
https://ardupilot.org/copter/docs/common-imu-fft-advanced-setup.html#common-imu-fft-advanced-setup

but the title in the above link writes that these steps are for using FFT data to run a Throttle Based HNTCH and not  for FFT based HNTCH setups

Then information of how your `FTN1.PkAvg`  should look like in page :

https://ardupilot.org/copter/docs/common-imu-fft-test-flight.html

is not descriptive on how large copters actually visualize also it doesnt tell you for example that in order to get the graph of second image you need to `mavfft_isb.py "/home/sok//logs/best_log_ever.bin"` or even hint you  to `ctrl+f `and hit the `FFT ` in order to visualize that .

What i was proposing was a tiny restructure on main FFT based page with a couple of gotchas for large copters and some tips like mavfft_isd and mavgraph .

Thank you for taking the time to answer to my issue,
Best!

**Comment by Hwurzburg** [comment]
feel free to propose changes or expansions....either as a text document, using the wiki direct editing button, or a normal PR to the wiki....we will review and can help with any  formatting, etc. issue...

**Comment by firebblade** [comment]
Thank you , 

ill try the `edit on girhub ` button .

Best!

**Comment by Hwurzburg** [comment]
I have re-orgd the notch pages....hopefully its clearer now...closing
